telegraf/content.md

```diff
@@ -14,99 +14,100 @@ Telegraf is an open source agent written in Go for collecting metrics and data o
 -	8092 UDP
 -	8094 TCP
 
-### Using default config
+### Using the default configuration
 
 The default configuration requires a running InfluxDB instance as an output plugin. Ensure that InfluxDB is running on port 8086 before starting the Telegraf container.
 
 Minimal example to start an InfluxDB container:
 
 ```console
-$ docker run -p 8083:8083 -p 8086:8086 influxdb
+$ docker run -d --name influx -p 8083:8083 -p 8086:8086 influxdb
```

Comment:
Can you change this name to `influxdb`? I think it's referred to that way in the `influxdb` container and I'd like to keep it consistent.